.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
@vidyanambiar did you intend to check in this vscode config?

Reply:
@florkbr Yes - I thought it would make it easy to debug in the future. If it isn't helpful, I can pull it out. LMK